sample/src/main/java/com/google/accompanist/sample/navigation/material/BottomSheetNavSample.kt

```diff
@@ -101,7 +116,17 @@ private fun HomeScreen(showSheet: () -> Unit, showFeed: () -> Unit) {
 
 @Composable
 private fun BottomSheet(showFeed: () -> Unit, showAnotherSheet: () -> Unit, arg: String) {
+    var items by remember { mutableStateOf(2) }
```

Comment:
Will undo the sample changes before merging :)

Reply:
ACK

Reply:
Is it worth having multiple samples to test specific cases?

Reply:
Not specifically samples I think, but I expanded tests to cover this (`testSheetContentSizeChangeDuringAnimation_opensSheet_shortSheetToShortSheet`, `testSheetContentSizeChangeDuringAnimation_opensSheet_shortSheetToTallSheet`, `testSheetContentSizeChangeDuringAnimation_opensSheet_tallSheetToTallSheet`, `testSheetContentSizeChangeDuringAnimation_opensSheet_tallSheetToShortSheet`)